server.go

```diff
@@ -71,6 +72,35 @@ type bug struct {
 	PointValue int    `json:"pointValue"`
 }
 
+type scoring_alert struct {
+	RecentHits []string `json:"recent_hits"` // encoded scoring message
+}
+
+type scoring_message struct {
```

Comment:
I think snake case is kinda "not a golang thing". I'd do `scoringMessage` or `ScoringMessage` if you need it to be public (it would be private to main in this current convention)

Reply:
done 22dcf90